spec/site/tlc/alarm_spec.rb

```diff
@@ -111,6 +111,57 @@ def verify_timestamp alarm, duration=1.minute
     # 5. When we resume the alarm
     # 6. Then we should receive an alarm resumed message
 
+    # Validate that alarm timestamps are different when alarm turns active vs inactive.
+    #
+    # This test verifies RSMP 3.1.4 behavior where the alarm timestamp (aTs) 
```

Comment:
if this behaviour differs between core versions, we need to tag the test with the appropriate core version, and maybe have more than one test